src/Serializer/composer.json

```diff
@@ -25,7 +25,6 @@
         "php": ">=8.2",
         "api-platform/metadata": "^3.4 || ^4.0",
         "api-platform/state": "^3.4 || ^4.0",
-        "doctrine/collections": "^2.1",
```

Comment:
Good catch. This must be added in `require-dev`.

Reply:
I update when i reopen my laptop